lib/ssh_scan/rule_engine.rb

```diff
@@ -0,0 +1,94 @@
+module SSHScan
+  class RuleEngine
+    # Simple one rule matching.
+    def self.matchSingleRule(key, value, result)
+      answer = true
```

Comment:
Should the default answer be false and not true?

I'd rather fall back than incorrectly match.